kpm/spec/kpm/unit/account_spec.rb

```diff
@@ -0,0 +1,347 @@
+      end
+
+      it 'when there is already payment method and skip payment method is true' do
+        expect(account_class.send(:process_import_data, cols_data, 'payment_methods', cols_names.split('|'), true, ['h'])).to be_nil
+      end
+    end
+
+  end
+
+  describe '#import_data' do
+    include_context 'account'
+
+    context 'when data to import; main import method' do
+      it 'when importing data with empty file' do
+        File.new(dummy_data_file, 'w+').close
+        expect{account_class.import_data(dummy_data_file,false,nil,true,false) }.to raise_error(Interrupt,"Data on #{dummy_data_file} is invalid")
+        File.delete(dummy_data_file)
+      end
+      it 'when importing data with no file' do
+        expect{account_class.import_data(dummy_data_file,false,nil,true,false) }.to raise_error(Interrupt,'Need to specify a valid file')
+      end
+      it 'when importing data with new record_id' do
+        open (dummy_data_file), 'w' do |io|
+          io.puts(dummy_data)
+        end
+        expect{account_class.import_data(dummy_data_file,false,nil,true,false) }.not_to raise_error(Interrupt)
+        response = `#{mysql_cli} "#{mysql_delete_dummy_account}" 2>&1`
+        expect(response.split("\n")[1]).to eq('1')
+        response = `#{mysql_cli} "#{mysql_delete_dummy_account_history}" 2>&1`
+        expect(response.split("\n")[1]).to eq('1')
+      end
+
+      it 'when importing data reusing record_id' do
+        open (dummy_data_file), 'w' do |io|
+          io.puts(dummy_data)
+        end
+        expect{account_class.import_data(dummy_data_file,true,nil,true,false) }.not_to raise_error(Interrupt)
+        response = `#{mysql_cli} "#{mysql_delete_dummy_account}" 2>&1`
+        expect(response.split("\n")[1]).to eq('1')
+        response = `#{mysql_cli} "#{mysql_delete_dummy_account_history}" 2>&1`
+        expect(response.split("\n")[1]).to eq('1')
+      end
+
+      it 'when importing data with different tenant_record_id' do
+        open (dummy_data_file), 'w' do |io|
+          io.puts(dummy_data)
+        end
+        expect{account_class.import_data(dummy_data_file,false,10,true,false) }.not_to raise_error(Interrupt)
+        response = `#{mysql_cli} "#{mysql_delete_dummy_account}" 2>&1`
+        expect(response.split("\n")[1]).to eq('1')
+        response = `#{mysql_cli} "#{mysql_delete_dummy_account_history}" 2>&1`
+        expect(response.split("\n")[1]).to eq('1')
+      end
+
+      it 'when round trip' do
+        open (dummy_data_file), 'w' do |io|
+          io.puts(dummy_data)
+        end
+        expect{account_class.import_data(dummy_data_file,false,10,true,true) }.not_to raise_error(Interrupt)
+        new_account_id = account_class.instance_variable_get(:@tables_id)
+
+        response = `#{mysql_cli} "DELETE FROM accounts WHERE id = '#{new_account_id['accounts_id']}'; SELECT ROW_COUNT();" 2>&1`
+        expect(response.split("\n")[1]).to eq('1')
```

Comment:
I think you need to pass the password as `MYSQL_PWD` (see https://dev.mysql.com/doc/refman/5.5/en/password-security-user.html) instead of the command line.

On my setup, this assertion fails because of the leading warning message:

```
"mysql: [Warning] Using a password on the command line interface can be insecure.\nROW_COUNT()\n1\n"
```